### PR TITLE
add Model

### PR DESCRIPTION
**Pull request by dinhhoa1994**
ready

### CODE REVIEW

**Comment by dinhhoa1994** [comment]
ready

File: app/Task.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model {
+	/**
+	 * The attributes that are mass assignable.
+	 *
+	 * @var array
+	 */
+	protected $fillable = ['name'];
```
Comment (quoctd-0607) [inline app/Task.php:13]: protected $fillable = [
'name',
];

**Comment by dinhhoa1994** [comment]
ready

File: app/Task.php (hunk 1 above)
Comment (dinhhoa1994) [reply app/Task.php]: ready

**Comment by dinhhoa1994** [comment]
read

**Comment by dinhhoa1994** [comment]
ready

**Comment by quoctd-0607** [comment]
merged

**Pull request merged**